src/bindings/python/src/openvino/frontend/__init__.py

```diff
@@ -39,10 +39,13 @@
 from openvino.pyopenvino import FrontEndManager
 from openvino.pyopenvino import FrontEnd
 from openvino.pyopenvino import InputModel
+from openvino.pyopenvino import NodeContext
 from openvino.pyopenvino import Place
 from openvino.pyopenvino import TelemetryExtension
 from openvino.pyopenvino import DecoderTransformationExtension
 from openvino.pyopenvino import JsonConfigExtension
+from openvino.pyopenvino import ConversionExtensionBase
+from openvino.pyopenvino import ConversionExtension
```

Comment:
nit: divide imports into two groups (Frontend routines, Extensions routines) and sort them within a group

Reply:
done